tests/opentelemetry-docker-tests/tests/sqlalchemy_tests/mixins.py

```diff
@@ -242,4 +243,6 @@ def insert_players(session):
                 close_all_sessions()
 
         spans = self.memory_exporter.get_finished_spans()
-        self.assertEqual(len(spans), 5)
+        self.assertEqual(
+            len(spans), 5 if self.VENDOR not in ["postgresql"] else 3
```

Comment:
why did this change in this PR? Can we document the difference between vendors as a comment?

Reply:
Added a comment. It's an optimisation that's enabled by default in 1.4 See https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#orm-batch-inserts-with-psycopg2-now-batch-statements-with-returning-in-most-cases